test/test_torch.py

```diff
@@ -5190,6 +5190,27 @@ def test_dot_data_use(self):
             model.weight.data = weight
             out = model(input)
 
+    def test_cpp_stacktraces(self):
+        python_error_msg = "Trying to create tensor with negative dimension"
+        cpp_stack_msg = "Exception raised from check_size_nonnegative at"
```

Comment:
This seems pretty sensitive to implementation details of `torch.rand`.

Reply:
It is, but I did not find a more generic way to check that the cpp stack-trace in the error message.
I'll add a comment explaining this here.